banzai/tests/test_calibration_stacker.py

```diff
@@ -59,7 +59,7 @@ def test_stacking_with_noise():
 
 
 def test_stacking_with_different_pixels():
-    d = np.arange(nx*ny, dtype=np.float).reshape(ny, nx)
+    d = np.arange(nx*ny, dtype=np.float64).reshape(ny, nx)
```

Comment:
Numpy has deprecated np.float in later versions, so be explicit here.

Reply:
I might advocate to just use `float` rather than np.float64. Also, don't we use float32 a lot of places?